packages/sysutils/busybox/scripts/init

```diff
@@ -383,8 +378,7 @@ load_splash() {
     if [ -f "$MODULE_DIR/uvesafb.ko" -a ! -e /dev/fb0 ]; then
       progress "Loading kernel module uvesafb.ko"
       insmod "$MODULE_DIR/uvesafb.ko" && \
-        set_default_res=yes || \
-        progress "... Failed to load kernel module uvesafb, skipping"
+        set_default_res=yes || progress "... Failed to load kernel module uvesafb, skipping"
```

Comment:
Not sure about this one - make it all one line (` ... && ... || ...`) or keep as 3 lines?

Reply:
Reduced to one line.